PokeDemo/Controllers/PokemonListViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+final class PokemonListViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet private weak var tableView: UITableView! {
+        didSet {
+            tableView.register(UINib(nibName: PokemonListCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: PokemonListCell.reuseIdentifier)
+            tableView.dataSource = self
+        }
+    }
+    
+    // MARK: - Properties
+    
+    // MARK: - Private
+    
+    private let presenter = PokemonListPresenter()
+    private var pokemons: [Pokemon]? = [Pokemon]()
```

Comment:
```suggestion
    private var pokemons: [Pokemon] = []
```

Reply:
Covered in c3f06092d44508dfe9c0782dff1248c27cb2cd05